core/controllers/base_test.py

```diff
@@ -939,6 +939,14 @@ def _get_tags(
             msg='Invalid HTML: %s at %s in %s' % (input_string, key, filename))
         return sorted(result)
 
+    def test_i18n_keys_format(self) -> None:
+        """Tests that the keys are correctly formatted.
+        """
```

Comment:
Please move this to the end of the previous line.

Reply:
Do you mean the """? I moved it to the end of line 943.